### PR TITLE
Localisation updates from https://translatewiki.net.

### PR DESCRIPTION
**Pull request by translatewiki**
Translation updates

### CODE REVIEW

**Comment by kelson42** [comment]
@MohitMaliDeveloper Why have you fixed something manually here? This is « forbidden »!

**Comment by MohitMaliFtechiz** [comment]
> @MohitMaliDeveloper Why have you fixed something manually here? This is « forbidden »!

@kelson42  I just did so that atleast in a month translate wiki changes were merged . But yes onces changes are done on upstream we do not required to do this.

**Comment by kelson42** [comment]
@MohitMaliFtechiz Does that mean you have still modifications made on TW waiting approval since two weeks?

**Comment by MohitMaliFtechiz** [comment]
> @MohitMaliFtechiz Does that mean you have still modifications made on TW waiting approval since two weeks?

Yes one changes are left to be approved there.

**Comment by kelson42** [comment]
@MohitMaliFtechiz Ok, we are not in a hurry, but if it takes months, this is going to be a problem. Please wait a bit, and if nothing happens you should write a message on TW to one of rhe other active translators in this language to explicitly request their help to validate your modification.

**Comment by kelson42** [comment]
@MohitMaliDeveloper Any news here? Are we OK now with TW (all the tickets have been closed)? Can we merge (still without modifying manually the PR)? If "not" please open corresponding bug tickets.

**Comment by kelson42** [comment]
@MohitMaliFtechiz why I don’t have a feedback after 3 days? As a reminder: handling already open PR is PRIO1.

**Comment by kelson42** [comment]
@MohitMaliFtechiz good i will check all of this later.

File: core/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -179,7 +181,7 @@
   <string name="your_languages">Langues sélectionnées :</string>
   <string name="other_languages">Autres langues :</string>
   <string name="no_items_msg">Aucun élément disponible</string>
-  <string name="crash_title">Hé bien&#8230; c’est embarrassant.</string>
+  <string name="crash_title">Hé bien.... c’est embarrassant.</string>
```
Comment (kelson42) [inline core/src/main/res/values-fr/strings.xml:184]: @MohitMaliFtechiz No reason to have `....` when in the original this is `...`

**Review by kelson42 — CHANGES_REQUESTED** [review]
Still NOK, I´m out of patience.

File: core/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (kelson42) [reply core/src/main/res/values-fr/strings.xml:184]: This is wrong and should be changed upstream in TW

**Pull request merged**